src/backend/utils/html/index.js

```diff
@@ -42,7 +44,10 @@ module.exports = function process(html) {
   lazyLoad(dom);
   // Replace <code> elements with encoded entities to use characters
   replaceCodeEntities(dom);
-
+  // Remove empty <p> elements
+  removeEmptyParagraphs(dom);
   // Return the resulting HTML
```

Comment:
This comment doesn't match what you're doing on the next line.  Move this comment to 51 above the final return statement, and properly document what you're doing with this code.